go.mod

```diff
@@ -142,3 +142,6 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
+
+// Temporary raplcement for testing https://github.com/lima-vm/go-qcow2reader/pull/36
+replace github.com/lima-vm/go-qcow2reader v0.2.1 => github.com/nirs/go-qcow2reader v0.0.0-20241025131821-46183d3fe6b2
```

Comment:
Released https://github.com/lima-vm/go-qcow2reader/releases/tag/v0.3.0

Reply:
go.mod was not updated yet, should I updated it in this PR?

Reply:
Add a commit to updated the dependency, can be removed if we go with the standard dependency update.